docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/postgres_data:/var/lib/postgresql/data
```

Comment:
It's a good practice add health-check in each docker service that is required by another.
Then, add something like:
`healthcheck: test: ["CMD", "pg_isready"] interval: 10s timeout: 5s retries: 3`

Reply:
Very nice. I didnt't know about `healthcheck` command in docker-compose. I will change it.

Reply:
no problem, any doubts contact @rafoliveira on opensanca's slack

Reply:
😂